<launch>
    <arg name="tree" default="game_tree"/>
    <node name="main_tree_node" pkg="vizzy_behavior_trees" type="main_tree_node" output="screen">
        <param name="bt_xml" value="$(find vizzy_behavior_trees)/example_trees/$(arg tree).xml"/>
    </node>
</launch>
